restcomm/restcomm.testsuite/src/test/java/org/mobicents/servlet/restcomm/http/AccountsEndpointTest.java

```diff
@@ -77,11 +81,29 @@ public void testCreateAccount() {
     }
 
     @Test
+    public void testCreateAccountCheckClientExisted() throws ClientProtocolException, IOException {
+        String subAccountPassword = "mynewpassword";
+        String subAccountEmail = "lyhungthinh@gmail.com";
+
+        if (!accountUpdated){
+            RestcommAccountsTool.getInstance().updateAccount(deploymentUrl.toString(), adminUsername, adminAuthToken,
+                    adminUsername, newAdminPassword, adminAccountSid, null);
+        }
+        JsonObject subAccountResponse = RestcommAccountsTool.getInstance().createAccount(deploymentUrl.toString(),
+                adminUsername, newAdminAuthToken, subAccountEmail, subAccountPassword);
+
+        JsonObject clientOfAccount = CreateClientsTool.getInstance()
+                                      .getClientOfAccount(deploymentUrl.toString(), subAccountResponse);
+        assertNotNull(clientOfAccount);
```

Comment:
You should also test that sip client can register (see https://github.com/RestComm/RestComm-Core/blob/2309d023db1b7636ad67926a3910397019f06a1a/restcomm/restcomm.testsuite/src/test/java/org/mobicents/servlet/restcomm/telephony/RegisterClientTest.java)